pkg/data/management/kontainerdriver_data.go

```diff
@@ -76,6 +76,9 @@ func addKontainerDrivers(management *config.ManagementContext) error {
 	}
 
 	creator.deleteRKEKontainerDriver()
```

Comment:
We also want to delete this code as well right ? 

```
if err := creator.add("googleKubernetesEngine"); err != nil {
		return err
	}
```

Reply:
yes